src/OrbitQt/TargetLabel.cpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "TargetLabel.h"
+
+namespace orbit_qt {
+
+namespace fs = std::filesystem;
+
+const QString TargetLabel::kDefaultColor{"white"};
+const QString TargetLabel::kGreenColor{"#66BB6A"};
+const QString TargetLabel::kOrangeColor{"orange"};
+const QString TargetLabel::kRedColor{"red"};
+const QString kStyleSheetFormat = "color: %1;";
+
+TargetLabel::TargetLabel(QWidget* parent) : QLabel(parent) {}
+
+void TargetLabel::SetFileTarget(const fs::path& path) {
+  setText(QString::fromStdString(path.filename().string()));
+  setStyleSheet(kStyleSheetFormat.arg(kDefaultColor));
+}
+
+void TargetLabel::SetStadiaTarget(const ProcessData& process, const orbit_ggp::Instance& instance) {
+  setText(
+      QString{"%1 @ %2"}.arg(QString::fromStdString(process.name())).arg(instance.display_name));
+  SetProcessAlive();
+}
+
+void TargetLabel::SetLocalTarget(const ProcessData& process) {
+  setText(QString{"%1 @ localhost"}.arg(QString::fromStdString(process.name())));
+  SetProcessAlive();
+}
+
+void TargetLabel::SetProcessDead() {
+  setStyleSheet(kStyleSheetFormat.arg(kOrangeColor));
+  setToolTip("The process ended on the instance");
```

Comment:
Nit: It's a sentence, I would make a fullstop at the end.

Reply:
done